app/src/main/java/dk/blackdarkness/g17/cphindustries/dataaccess/SceneDaoDemo.java

```diff
@@ -37,15 +37,15 @@ public Scene get(int sceneId) {
 
     @Override
     public void create(Scene scene) {
-        scene.setId(0);
+        scene.setId(1);
 
         this.allScenes = DemoDataRepository.loadListOfScenes();
 
         // Finds new sceneId
         for (Scene s : allScenes) {
-            if (s.getId() > scene.getId()) {
+            if (s.getId() == scene.getId()) {
                 scene.setId(s.getId() + 1);
```

Comment:
Ville det ikke være pænere at have `continue` i if-statement i stedet for `else break;`?

Reply:
Nope.
`continue` gør præcis det den antyder, nemlig at fortsætte loop'et. Det er ikke det vi vil her, vi vil gerne "break'e" ud af loop'et, da vi har fundet et tomt ID, og dermed gerne vil gå videre i metoden.

Reply:
Jeg ved ikke hvorfor jeg foreslog det der, fordi det giver slet ikke mening.

Det problem der er dog; listen er ikke nødvendigvis sorteret efter `id`